### PR TITLE
bazel: bundlesize check

### PR DESCRIPTION
**Pull request by valerybugakov**
## Context

Enabled the bundlesize check in Bazel.

## Test plan

CI and see that the bundlesize check in GitHub actions.

### CODE REVIEW

**Comment by sg-e2e-regression-test-bob** [comment]
## Bundle size report 📦

| Initial size | Total size | Async size | Modules |
| --- | --- | --- | --- |
| 0.54% (+14.68 kb) 🔺 | 0.15% (+22.21 kb) 🔺 | 0.06% (+7.53 kb) | 0.00% (0) |

Look at the [Statoscope report](https://console.cloud.google.com/storage/browser/_details/sourcegraph_reports/statoscope-reports/vb/bazel-bundlesize-check/compare-report.html;tab=live_object) for a full comparison between the commits 930485a and 185c154 or [learn more](https://docs.sourcegraph.com/dev/how-to/testing#bundlesize).


<details>
<summary>Open explanation</summary>

- `Initial size` is the size of the initial bundle (the one that is loaded when you open the page)
- `Total size` is the size of the initial bundle + all the async loaded chunks
- `Async size` is the size of all the async loaded chunks
- `Modules` is the number of modules in the initial bundle
</details>

File: enterprise/dev/ci/internal/ci/operations.go [hunk 1]
```diff
@@ -286,7 +286,7 @@ func addWebAppEnterpriseBuild(opts CoreTestOperationsOptions) operations.Operati
 			bk.Env("ENTERPRISE", "1"),
 			bk.Env("CHECK_BUNDLESIZE", "1"),
 			// To ensure the Bundlesize output can be diffed to the baseline on main
-			bk.Env("WEBPACK_USE_NAMED_CHUNKS", "true"),
+			// bk.Env("WEBPACK_USE_NAMED_CHUNKS", "true"),
```
Comment (burmudar) [inline enterprise/dev/ci/internal/ci/operations.go:289]: Should this be commented?

File: client/web/BUILD.bazel [hunk 2]
```diff
@@ -1978,6 +1979,7 @@ webpack_bundle(
     ],
     entry_points = {
```
Comment (burmudar) [inline client/web/BUILD.bazel:1980]: Oh Nice! I didn't know it could be  a dictionary! What does "app" and "embed" do?

**Review by burmudar — APPROVED** [review]

File: enterprise/dev/ci/internal/ci/operations.go (hunk 1 above)
Comment (valerybugakov) [reply enterprise/dev/ci/internal/ci/operations.go:289]: Can drop this 👍

File: client/web/BUILD.bazel (hunk 2 above)
Comment (valerybugakov) [reply client/web/BUILD.bazel:1980]: It means we're building two "applications" — our usual web application and a smaller one used to serve embedded notebooks.

The embed route is used to serve embeddable content (via an iframe) to 3rd party sites. Example iframe tag: `<iframe src="https://sourcegraph.com/embed/notebooks/123" frameborder="0" sandbox="allow-scripts allow-same-origin allow-popups"></iframe>`

Comment (burmudar) [reply client/web/BUILD.bazel:1980]: Ah Awesome. TIL!

**Pull request merged**